exatomic/molcas/output.py

```diff
@@ -199,7 +198,7 @@ def add_orb(self, path, mocoefs='coef', orbocc='occupation'):
     def add_overlap(self, path):
         try: # If it's an ASCII text file
             self.overlap = Overlap.from_column(path)
-        except ParserError: # If it's an HDF5 file
+        except Exception: # If it's an HDF5 file
```

Comment:
Is `ParserError` deprecated? Is there a proper replacement - like maybe `pandas.errors.ParserError`?